blivet/devices/lvm.py

```diff
@@ -1789,8 +1789,132 @@ def populate_ksdata(self, data):
         data.pool_name = self.pool.lvname
 
 
+class LVMVDOPoolMixin(object):
```

Comment:
Is it not useful/possible to derive from `LVMThinPoolMixin` to save some of the boilerplate?